src/schemas/validation/schema.yaml

```
@@ -230,6 +230,24 @@ $defs:
         type: array
         items:
           $ref: '#/$defs/parameter-or-reference'
+      additionalOperations:
+        type: object
+        additionalProperties:
+          $ref: '#/$defs/operation'
+        propertyNames:
+          $comment: RFC9110 restricts methods to "1*tchar" in ABNF
+          pattern: "^[a-zA-Z0-9!#$%&'*+.^`|~-]+$"
```

I thought I committed this but apparently not?  Trying again.

```suggestion
          pattern: "^[a-zA-Z0-9!#$%&'*+.^`|~]+$-"
```

---

You changed it once to move the hyphen the end of the character class (correct) but then moved it to the end of the regex (not correct)

---

:facepalm: